eng/testing/linker/trimmingTests.targets

```diff
@@ -81,14 +81,17 @@
       <_additionalPropertiesString>@(_propertiesAsItems->'&lt;%(Identity)&gt;%(Value)&lt;/%(Identity)&gt;', '%0a    ')</_additionalPropertiesString>
     </PropertyGroup>
 
+    <!-- RunNativeAotTestApps trumps TestTrimming, same as PublishAot trumps PublishTrimmed -->
```

Comment:
(nit) this comment doesn't align with the code. The comment says one trumps the other. The code says if they are both true it is an error.